docker/docker.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+trap "echo; exit" INT
+trap "echo; exit" HUP
+
+# install `realpath` on macOS from Homebrew `coreutils`
+# reference: https://unix.stackexchange.com/a/336138/269147
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  set -e
+  printf "\n*** Mac OS (Darwin) detected."
+  printf "\n*** Installation of Homebrew and coreutils is required by this script."
+  printf "\n*** Press SPACE to continue the installation or Ctrl+C to exit..."
+  while true; do
+    read -n1 -r
+    [[ $REPLY == ' ' ]] && break
+  done
+  printf "\n*** Continuing..."
+
+  if ! which brew >/dev/null 2>&1; then
+    printf "\n*** Installing Homebrew..."
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+  fi
+
```

Comment:
```bash
  if ! which realpath >/dev/null 2>&1; then
    brew update
    printf "\n*** Installing coreutils..."
    brew install coreutils
  fi
```

Run `brew update` and `brew install coreutils` if `realpath` is not installed.

Reply:
i pushed another commit here for review https://github.com/a16z/helios/pull/262/commits/5554620dd066756654b8081783a30ad6dafa5de6